k8s/pkg/apis/longhorn/v1beta2/instancemanager.go

```diff
@@ -55,12 +55,13 @@ type InstanceProcessSpec struct {
 type InstanceState string
 
 const (
-	InstanceStateRunning  = InstanceState("running")
-	InstanceStateStopped  = InstanceState("stopped")
-	InstanceStateError    = InstanceState("error")
-	InstanceStateStarting = InstanceState("starting")
-	InstanceStateStopping = InstanceState("stopping")
-	InstanceStateUnknown  = InstanceState("unknown")
+	InstanceStateRunning    = InstanceState("running")
+	InstanceStateStopped    = InstanceState("stopped")
+	InstanceStateError      = InstanceState("error")
+	InstanceStateStarting   = InstanceState("starting")
+	InstanceStateStopping   = InstanceState("stopping")
+	InstanceStateUnknown    = InstanceState("unknown")
+	InstanceStateTerminated = InstanceState("terminated")
```

Comment:
Could you elaborate more about the use of the InstanceStateTerminated state?

Reply:
The `terminated` state represents that an instance was detected by the instance manager, but disappeared from the instance manager later. This is the trigger for orphan controller get notice that an orphan complete the cleanup. Will remove this because we're going to move the orphaned instance status update to orphan controller (see [below](https://github.com/longhorn/longhorn-manager/pull/3694#discussion_r2051284164))

Reply:
I see. Thank you